admin/scripts/generateExamples.js

```diff
@@ -194,9 +196,9 @@ const templates = (
   await fs.readdir('./packages/create-docusaurus/templates')
 ).filter((name) => !excludes.includes(name));
 console.log(`Will generate examples for templates: ${templates.join(',')}`);
-for (const template of templates) {
-  await generateTemplateExample(template);
-}
+await Promise.all(
+  templates.map((template) => generateTemplateExample(template))
+);
```

Comment:
This is very much intentionally serial, because `generateTemplateExample` contains logging, and the logging must be in sequence.